ark/phenotyping/run_pixel_som.R

```diff
@@ -48,19 +48,16 @@ for (i in 1:length(fovs)) {
     matPath <- file.path(pixelMatDir, fileName)
     fovPixelData <- arrow::read_feather(matPath)
 
-    # 99.9% normalize pixel data
+    # 99.9% normalize pixel data based on values computed for subsetted pixel data in train_pixel_som
```

Comment:
Sorry I couldn't add this comment above since those lines weren't changed recently, but at the very top of this file, where you have "Usage: ..." (line 3), {markers} should no longer be there.